client-app/pages/loyalty/loyalty.vue

```diff
@@ -0,0 +1,456 @@
+<template>
+  <div class="bg-gray-100 flex-grow pt-6 pb-16 shadow-inner">
+    <div class="max-w-screen-2xl md:px-12 mx-auto">
+      <div class="flex lg:space-x-5">
+        <!-- First column-->
+        <div class="hidden lg:flex flex-col lg:w-1/5 space-y-5">
+          <AccountNavigation></AccountNavigation>
+        </div>
+
+        <!-- Second column-->
+        <div class="flex flex-col w-full lg:w-4/5 space-y-5">
+          <div class="flex flex-col">
+            <VcCard :title="$t('pages.loyalty.points_operations.balance_card.title')">
+              <div class="flex" v-if="userBalanceLoading">
+                <span class="h-10 w-64 bg-gray-200 animate-pulse"></span>
+              </div>
+              <div class="flex gap-2" v-else>
+                <span class="text-4xl font-extrabold">{{ myUserBalance.amount }}</span>
+                <span class="text-4xl left-10">
+                  {{ $t("pages.loyalty.points_operations.balance_card.points_label", myUserBalance.amount) }}</span
+                >
+              </div>
+            </VcCard>
+          </div>
+
+          <div class="flex justify-between items-center mx-5 lg:mx-0">
+            <h2 class="text-gray-800 text-3xl font-bold uppercase">{{ title }}</h2>
+
+            <VcButton v-if="!editingMode" class="px-3 uppercase border" size="sm" is-outline @click="openEditMode()">
+              {{
+                isMobile
+                  ? $t("pages.loyalty.points_operations.operations_list.add_new_operation_button_mobile")
+                  : $t("pages.loyalty.points_operations.operations_list.add_new_operation_button")
+              }}
+            </VcButton>
+          </div>
+
+          <div class="flex flex-col mx-5 lg:mx-0">
+            <VcCheckbox v-model="infiniteScrollEnabled" @change="onScrollModeChange()">
+              {{ $t("pages.loyalty.points_operations.operations_list.use_infinite_scrolling") }}
+            </VcCheckbox>
+          </div>
+
+          <div class="flex flex-col bg-white shadow-sm" :class="{ 'rounded border': !isMobile }">
+            <PointsOperationForm
+              v-if="editingMode"
+              :model-value="newOperation"
+              :disabled="saveOperationLoading"
+              class="px-6 py-4"
+              @save="saveOperation"
+            >
+              <template #append="{ dirty }">
+                <div class="flex space-x-4 pb-3 pt-7 sm:pb-4 sm:pt-4 sm:float-right">
+                  <VcButton
+                    kind="secondary"
+                    :size="isMobile ? 'md' : 'lg'"
+                    :is-disabled="saveOperationLoading"
+                    class="uppercase w-32 sm:w-auto sm:px-12"
+                    is-outline
+                    @click="closeEditMode"
+                    v-t="'pages.loyalty.points_operations.operations_list.cancel_button'"
```

Comment:
Please, use `v-t` directive only on plain HTML entities. `v-t` directive overrides any component inner structure. So if the component contains any slots or complex inner logic it will be overriden. Use `{{ $t('...') }}` instead.